pyrit/datasets/fetch_example_datasets.py

```diff
@@ -420,6 +420,20 @@ def fetch_pku_safe_rlhf_dataset(include_safe_prompts: bool = True) -> PromptData
 
     return dataset
 
+def fetch_harmful_hf_dataset() -> PromptDataset:
+    data = load_dataset("LLM-LAT/harmful-dataset", "default")
+
+    prompts = [item["prompt"] for item in data["train"]]
+
+    dataset = PromptDataset(
+        name="LLM-LAT/harmful-dataset",
+        source="https://huggingface.co/datasets/LLM-LAT/harmful-dataset",
+        harm_category='',  
+        description='harmful prompts',  
+        prompts=prompts,
+        should_be_blocked=False
```

Comment:
```suggestion
        should_be_blocked=True
```
Not super important as this field will vanish shortly but with harmful prompts we typically want to have them blocked.